Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -52,7 +52,7 @@ python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --
 * `--cephfs-metadata-pool-name`: (optional) Provides the name of the cephfs metadata pool
 * `--cephfs-filesystem-name`: (optional) The name of the filesystem, used for creating CephFS StorageClass
 * `--cephfs-data-pool-name`: (optional) Provides the name of the CephFS data pool, used for creating CephFS StorageClass
-* `--rados-namespace`: (optional) Divides a pool into separate logical namespaces, used for creating RBD PVC in a RadosNamespaces
+* `--rados-namespace`: (optional) Divides a pool into separate logical namespaces, used for creating RBD PVC in a RadosNamespaces (should be lower case)
```

Comment:
Two points. This isn't plural, so shouldn't end in an S. But moreso, RadosNamespace seems half-specified. Should this be the fully-specified `CephBlockPoolRadosNamespace` (not plural)? Or maybe just `rados namespace`? I'd suggest fully specifying it for users' benefits.

```suggestion
* `--rados-namespace`: (optional) Divides a pool into separate logical namespaces, used for creating RBD PVC in a CephBlockPoolRadosNamespace (should be lower case)
```